pkg/kubectl/cmd/run.go

```diff
@@ -136,7 +136,7 @@ func addRunFlags(cmd *cobra.Command) {
 	cmd.Flags().String("schedule", "", i18n.T("A schedule in the Cron format the job should be run with."))
 }
 
-func Run(f cmdutil.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer, cmd *cobra.Command, args []string, argsLenAtDash int) error {
+func RunRun(f cmdutil.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer, cmd *cobra.Command, args []string, argsLenAtDash int) error {
```

Comment:
Correct. I'd say a good majority of the exported symbols under `/kubectl/` could be made private. I did not want to create too much noise in this diff.

If I hide more of these identifiers I will isolate the rename commits to a single new PR.